docs/source/blog/gsoc2025.md

```diff
@@ -0,0 +1,47 @@
+:blogpost: true
+:date: February 28, 2025
+:author: Sofía Miñano
+:location: London, UK
+:category: Blog
+:language: English
+:image: 1
+
+(target-gsoc2025)=
+# We are participating in Google Summer of Code 2025!
+*Join us over the summer and contribute to open-source neuroscience tools*
+
+```{image} /_static/blog_images/gsoc2025/GSoC-icon.svg
+:width: 40%
+```
+```{image} /_static/logo_light.png
+:width: 33%
```

Comment:
On my browser, these don't look great. They're different sizes, and weirdly large. Not sure how to improve it, could just remove?

Reply:
@niksirbi did his magic and created a single image instead - would you mind having a look? I think it will behave much better now

Reply:
Perfect